lib/elixir/lib/map_set.ex

```diff
@@ -121,7 +124,7 @@ defmodule MapSet do
       #MapSet<[1, 3]>
 
   """
-  @spec delete(t, value) :: t
+  @spec delete(t(val), val) :: t(val) when val: term
```

Comment:
Ops, I actually missed this as I started with `val: term` everywhere then moved to `val: value` everywhere except here 😄 Thanks!